src/main/java/org/opentripplanner/routing/error/NoStopsInRangeException.java

```diff
@@ -0,0 +1,21 @@
+package org.opentripplanner.routing.error;
+
+import java.util.List;
+
+/**
+ * Indicates that a vertex was found, but no stops could be found within the search radius.
+ */
+public class NoStopsInRangeException extends RuntimeException {
+  private static final long serialVersionUID = 1L;
+
+  List<String> missing;
+
+  public NoStopsInRangeException(List<String> missing) {
+    super("vertices not found: " + missing.toString());
+    this.missing = missing;
+  }
+
+  public List<String> getMissing() {
+    return missing;
+  }
+}
```

Comment:
We do not need to distinguish between different error situation, so this exception should be more general like e.g. `RoutingValidationException` and include the RoutingError. Then maybe the error handling in RoutingWorker collapses to one line and you do not need to make the `routingErrors` class scoped (witch is not ideal).

Reply:
I have refactored the code to use the RoutingValidationException with a RoutingError included. This also meant I could delete som of the other Exception types.